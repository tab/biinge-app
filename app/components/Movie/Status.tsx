import React from "react"
import { View, Text } from "react-native"

import { statusStyles } from "styles"

type Props = {
  children: React.ReactNode
}

const StatusComponent = ({ children }: Props) => {
  return (
    <View style={statusStyles.container}>
      <Text style={statusStyles.text}>{children}</Text>
    </View>
  )
}

export default StatusComponent
