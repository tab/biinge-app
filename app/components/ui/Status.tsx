import React from "react"
import { View, Text } from "react-native"

import { statusStyles } from "styles"

type Props = {
  children: React.ReactNode
}

const StatusComponent = ({ children }: Props) => {
  return (
    <View style={statusStyles.root}>
      <Text style={statusStyles.text}>{children}</Text>
    </View>
  )
}

export default StatusComponent
