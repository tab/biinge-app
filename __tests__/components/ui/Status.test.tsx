import React from "react"
import "react-native"
import { render } from "@testing-library/react-native"

import { it } from "@jest/globals"

import Status from "components/ui/Status"

describe("Status", () => {
  it("renders correctly", () => {
    const tree = render(<Status>Post Production</Status>)
    expect(tree).toMatchSnapshot()
  })
})
