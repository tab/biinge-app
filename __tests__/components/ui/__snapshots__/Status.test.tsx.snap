// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "borderColor": "#2b2835",
      "borderRadius": 50,
      "borderWidth": 2,
      "flexDirection": "row",
      "justifyContent": "center",
      "paddingBottom": 2,
      "paddingHorizontal": 12,
    }
  }
>
  <Text
    style={
      {
        "color": "#2b2835",
        "fontSize": 12,
        "fontWeight": "700",
        "lineHeight": 22,
      }
    }
  >
    Post Production
  </Text>
</View>
`;
